src/parser/ccdcmol2-parser.ts

```diff
@@ -83,8 +83,12 @@ function buildUnitCell(s: Structure, cellDimensions: string): void {
     unitcellDict.alpha = alpha;
     unitcellDict.beta = beta;
     unitcellDict.gamma = gamma;
+  
+    // Add name to unitcell
+    unitcellDict.spacegroup = cellDimensions;
   }
```

Comment:
Could we make it more obvious as to what those values are, or would those in the know, know? And does it need the CCDCMOL bit at the end? Maybe check with Seth?

Reply:
CCDCMOL is from the file parser so don't know if we can get rid of this. Umm the cell dimensions was requested by Seth so I guess its ok